app/src/main/res/layout/fragment_home.xml

```diff
@@ -50,6 +40,14 @@
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintTop_toTopOf="@+id/tvWelcomeUser" />
 
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/rvSubjects"
+            android:layout_width="409dp"
+            android:layout_height="400dp"
```

Comment:
layout should match parent and add layout margin. you can change the screen size and check if UI is scalable.